src/cli/credrank.js

```diff
@@ -1,20 +1,16 @@
 // @flow
 
-import fs from "fs-extra";
-import stringify from "json-stable-stringify";
-import {join as pathJoin} from "path";
 import {sum} from "d3-array";
 import {format} from "d3-format";
 
 import sortBy from "../util/sortBy";
-import {credrank} from "../core/credrank/compute";
+import {credrank as callCredrank} from "../api/credrank";
```

Comment:
nit: Why `callCredrank` rather than just `credrank`? All function calls
are calls; this just confuses me when I read it because I have to figure
out what it does that’s special (answer: nothing).

Reply:
It was causing a namespace conflict with the credrank cli function that it's being called from.